src/lib/Scenes/Search/__tests__/AutosuggestResults-tests.tsx

```diff
@@ -0,0 +1,175 @@
+import { Theme } from "@artsy/palette"
+import SwitchBoard from "lib/NativeModules/SwitchBoard"
+import { defaultEnvironment } from "lib/relay/createEnvironment"
+import { flushPromiseQueue } from "lib/tests/flushPromiseQueue"
+import { CatchErrors } from "lib/utils/CatchErrors"
+import React from "react"
+import { TouchableOpacity } from "react-native"
+import ReactTestRenderer, { act } from "react-test-renderer"
+import { createMockEnvironment, MockPayloadGenerator } from "relay-test-utils"
+import { AutosuggestResult, AutosuggestResults } from "../AutosuggestResults"
+
+const TestWrapper: typeof AutosuggestResults = props => (
+  <Theme>
+    <CatchErrors>
+      <AutosuggestResults {...props} />
+    </CatchErrors>
+  </Theme>
+)
+
+jest.mock("lib/relay/createEnvironment", () => ({
+  defaultEnvironment: require("relay-test-utils").createMockEnvironment(),
+}))
+
+jest.mock("lodash", () => ({
+  throttle: f => f,
+}))
+
+jest.unmock("react-relay")
+
+// tslint:disable-next-line:no-empty
+jest.mock("react-native-sentry", () => ({ captureMessage() {} }))
+
+jest.mock("lib/NativeModules/SwitchBoard", () => ({ presentNavigationViewController: jest.fn() }))
+
+const env = defaultEnvironment as ReturnType<typeof createMockEnvironment>
```

Comment:
This type exists in @types/relay-test-utils but it's not exported so I've made a note to fix that too.